### PR TITLE
Italian support

### PR DESCRIPTION
**Pull request by anb**
As discussed with same license as other part of the project.

### CODE REVIEW

File: num2words/__init__.py [hunk 1]
```diff
@@ -50,6 +51,7 @@
     'dk': lang_DK.Num2Word_DK(),
     'pt_BR': lang_PT_BR.Num2Word_PT_BR(),
     'he': lang_HE.Num2Word_HE()
```
Comment (hsoft) [inline num2words/__init__.py]: Sorry for the delay in the review.

This is invalid syntax. I suggest that you run [tox](https://pypi.python.org/pypi/tox) on the repo before pushing.

Comment (hsoft) [inline num2words/__init__.py]: (I know, the best solution would be for a CI system to be plugged in this repo so we wouldn't have to do that manually...)

**Comment by anb** [comment]
Sorry about the delay. I ran tox and fixed the syntax.

**Comment by erozqba** [comment]
@anb the tests are failing because I set up travis-ci again to run test for 4 different versions of python, all test pass for python 2.7 but fails for python 3. Could you please make your changes python 3 compatible?

**Comment by anb** [comment]
Here you go. Hopefully we're good to go now.

**Pull request merged**